c63086455.lua

```diff
@@ -50,3 +50,6 @@ function c63086455.activate(e,tp,eg,ep,ev,re,r,rp)
 		end
 	end
 end
+function c63086455.tgopfilter(c,e)
+	return  c:IsRelateToEffect(e) and c:IsLocation(LOCATION_MZONE)
```

Comment:
should use c:IsRelateToEffect(e) and c:IsLocation(LOCATION_ONFIELD)?

Reply:
yep